SConstruct

```diff
@@ -345,81 +343,43 @@ config_options = [
         """Cantera needs to know where to find the Python interpreter. If
            PYTHON_CMD is not set, then the configuration process will use the
            same Python interpreter being used by SCons.""",
-        sys.executable),
+        sys.executable, PathVariable.PathAccept),
     PathVariable(
         'python_array_home',
         """If NumPy was installed using the '--home' option, set this to the home
-           directory for NumPy for Python 2.""",
+           directory for NumPy.""",
         '', PathVariable.PathAccept),
     PathVariable(
         'python_prefix',
-        """Use this option if you want to install the Cantera Python 2 package to
+        """Use this option if you want to install the Cantera Python package to
            an alternate location. On Unix-like systems, the default is the same
            as the 'prefix' option. If the 'python_prefix' option is set to
            the empty string or the 'prefix' option is not set, then the package
            will be installed to the system default 'site-packages' directory.
            To install to the current user's 'site-packages' directory, use
            'python_prefix=USER'.""",
         defaults.python_prefix, PathVariable.PathAccept),
-    EnumVariable(
```

Comment:
This PR is going to need a corresponding PR over at cantera-website to remove the config options and/or update the existing options, depending on how we want to handle documenting building the dev branch.

Reply:
Maybe a separate page with the options for the master branch? Something like,  if you're compiling a tag, go here. If you're compiling master, go here.

Reply:
New version of the docs with separate pages for both https://testing.cantera.org/compiling/configure-build-dev.html and https://testing.cantera.org/compiling/config-options-dev.html are up on the testing site.

Reply:
That looks good to me